packages/jaeger-ui/src/components/App/TraceIDSearchInput.tsx

```diff
@@ -36,7 +36,12 @@ class TraceIDSearchInput extends React.PureComponent<Props> {
 
   render() {
     return (
-      <Form layout="horizontal" onSubmit={this.goToTrace} className="TraceIDSearchInput--form">
+      <Form
+        aria-label="form"
```

Comment:
Form as HTML element is generally invisible to user, how does accessibility label help? I think it would make more sense on the `<input>` with label "trace ID"

Reply:
I don't think testing needs is a good reason to add (permanent) attributes to HTML if they don't improve the HTML itself. I would suggest finding other ways to locate elements for tests.

Reply:
Also, because form itself is invisible for the user, if some test is written that requires access to the form then the test probably does not pass the principles of testing established by RTL, "test from user perspective".

Reply:
> does not pass the principles of testing established by RTL, "test from user perspective".

Yes, for that reason only, E2E tests work best in frontend testing.